code/game/objects/items/storage/belt.dm

```diff
@@ -545,7 +545,7 @@
 /obj/item/storage/belt/wands/full/PopulateContents()
 	new /obj/item/gun/magic/wand/death(src)
 	new /obj/item/gun/magic/wand/resurrection(src)
-	new /obj/item/gun/magic/wand/polymorph(src)
+	new /obj/item/gun/magic/wand/fireball(src) //SKYEDIT EDIT - Trades polymorph for second fireball
```

Comment:
First, what's a "SKYEDIT".
Second, not sure if giving a second fireball wand is a good idea.

Reply:
What would you want to put there instead?
Given that they are both designed to be offensive, I think it fits.
The fireball wand is the most basic as well, you can even get it as lavaland loot.

Reply:
See, I don't know what would be a good alternative - that's the point. Spitballing right now, maybe uh.. an animation wand? Not sure if that's a thing, shouldn't be hard to add otherwise.
And yes, I'm very much aware you can get a fire wand from lavaland.

Also, yes. SKYEDIT sounds cool, but it makes searching "SKYRAT" to find our changes not a thing. *A certain brazilian coder* did that on the oldbase, and I still can't find every "Skyart" or whatever typo they made - nevermind the "skycommie"/whatever else.

Reply:
Actually wand of Animation sounds pretty funny.
If I get around to doing that, I'll add it in a new PR.
Because I would still have to sprite it and stuff.

Reply:
Epic.